src/ext.rs

```diff
@@ -0,0 +1,727 @@
+#[cfg(feature = "stm32f3")]
+pub use stm32f3::*;
+#[cfg(feature = "stm32g4")]
+pub use stm32g4::*;
+#[cfg(feature = "stm32h7")]
+pub use stm32h7::*;
+
+use crate::pac;
+use pac::hrtim_master as master;
+use pac::hrtim_tima as tima;
+use pac::hrtim_timb as timb;
+use pac::hrtim_timc as timc;
+use pac::hrtim_timd as timd;
+use pac::hrtim_time as time;
+#[cfg(feature = "hrtim_v2")]
+use pac::hrtim_timf as timf;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
```suggestion
#[derive(Clone, Copy, Debug, PartialEq, Eq)]
#[cfg_attr(feature = "defmt", derive(defmt::Format))]
```

Is it possible to derive `defmt::Format` for these?